src/adapters/supabase/helpers/client.ts

```diff
@@ -271,3 +271,9 @@ export const getWalletMultiplier = async (username: string): Promise<number> =>
   if (data?.multiplier == null) return 1;
   else return data?.multiplier;
 };
+
+export const getWalletReason = async (username: string): Promise<string> => {
```

Comment:
This should be `getMultiplierReason`

Reply:
https://github.com/ubiquity/ubiquibot/commit/1c14efc29cc15820fe900e491112570723ef72de